go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -722,6 +722,44 @@ func (fbo *folderBranchOps) isUnmergedLocked(lState *lockState) bool {
 	return fbo.unmergedBID != kbfsmd.NullBranchID
 }
 
+var errJournalNotAvailable = errors.New("could not get journal for TLF")
+
+// clearConflictView tells the journal to move any pending writes elsewhere,
+// resets the CR counter, and resets the FBO to have a synced view of the TLF.
+func (fbo *folderBranchOps) clearConflictView(ctx context.Context) error {
+	// TODO later: show the cleared conflict view under a special path,
+	//  so users can copy any unmerged files manually back into
+	//  their synced view before nuking it.
+
+	// TODO: I'm probably forgetting something
+	// just having a way to automatically rename the journal directory,
+	// and resetting the FBO to have the synced view of the folder,
+	// will be good enough.
+	journalServer, err := GetJournalServer(fbo.config)
+	if err != nil {
+		return err
+	}
+	journal, ok := journalServer.getTLFJournal(fbo.folderBranch.Tlf,
+		fbo.head.GetTlfHandle())
+	if !ok {
+		return errJournalNotAvailable
+	}
+	err = journal.moveAway(ctx)
+	if err != nil {
+		return err
+	}
+
+	err = fbo.cr.clearConflictRecords()
+	if err != nil {
+		return err
+	}
+	lState := makeFBOLockState()
+	fbo.mdWriterLock.Lock(lState)
+	defer fbo.mdWriterLock.Unlock(lState)
+	fbo.setBranchIDLocked(lState, kbfsmd.NullBranchID)
```

Comment:
This doesn't quite work, because you need to revert the cached MD head to the merged version as well.  But since that changes the view of the data, we also have to tell the kernel to invalidate any caches they have for that old data.

Luckily, there's already a function that does all this for you: `folderBranchOps.unstageLocked()`.  Though it invokes `PruneBranch`, which forces the journal server to delete its current journal state for the TLF, instead of moving it away.  So maybe you can just massage that function to take in a parameter that lets it call `moveAway` instead of `PruneBranch` if we are trying to clear conflicts in this way?

Another point is that you have to unstage all the commits first before you move the journal out of the way, and keep the `mdWriterLock` the whole time; otherwise new reads and writes could slip in and cause races.  But `unstagedLocked` should do all that correctly.

There might be one other option however, that's both a little more foolproof than unstaging, and would also easily allow the user to navigate to the file view of the conflict branch.  We could just rename the TLF to something else, like we do when the TLF name becomes conflicted or finalized.  It would require more work though to expose a name like that, especially after a restart, but it might be the right play in the long run.  If you want to go this route, we can talk more about it tomorrow.  But I'm fine with just using `unstageLocked` for now, and we can do the TLF namespace stuff later.